NETProvider/src/FirebirdSql.Data.UnitTests/SrpTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿/*
+ *  Firebird ADO.NET Data provider for .NET and Mono
+ *
+ *     The contents of this file are subject to the Initial
+ *     Developer's Public License Version 1.0 (the "License");
+ *     you may not use this file except in compliance with the
+ *     License. You may obtain a copy of the License at
+ *     http://www.firebirdsql.org/index.php?op=doc&id=idpl
+ *
+ *     Software distributed under the License is distributed on
+ *     an "AS IS" basis, WITHOUT WARRANTY OF ANY KIND, either
+ *     express or implied.  See the License for the specific
+ *     language governing rights and limitations under the License.
+ *
+ *  Copyright (c) 2002, 2007 Carlos Guzman Alvarez
+ *  All Rights Reserved.
+ */
+
+using System;
+using System.Data;
+using System.Numerics;
+using System.Data.Common;
+
+using FirebirdSql.Data.FirebirdClient;
+using FirebirdSql.Data.Client.Managed.Version10;
+using NUnit.Framework;
+
+namespace FirebirdSql.Data.UnitTests
+{
+	[TestFixture(FbServerType.Default)]
+	[TestFixture(FbServerType.Embedded)]
```

Comment:
Replace with non-parameterized TextFixture.
Is it correct ?
My NUnit seems 2.4.8 and seems it works.
https://github.com/nakagami/FirebirdSql.Data.FirebirdClient/commit/f60f45a13ad74380fa1372b631988d317e157a25#diff-83243b5369229d9fdb8088741d8bba89L30